Using import/export syntax in addon-test-support causing errors in eslint

Copy from https://github.com/ember-cli/ember-addon-output/issues/6

I've tried to create test helpers for my addon but eslint throws errors because `addon-test-support` folder uses `sourceType: 'script`. Adding `'addon-test-support/**'` to my eslintrc fixes it. 

https://github.com/ember-cli/ember-addon-output/blob/82451684f5bd92f77ccc0c02095cf2af5de647a9/.eslintrc.js#L29-L33

I can submit PR for this just let me know.

https://github.com/ember-cli/ember-cli/blob/66015518df097cf5d649c989115675805d0b573b/blueprints/app/files/.eslintrc.js#L29-L34

Error:
```
  1:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

✖ 1 problem (1 error, 0 warnings)
```

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.0.0
http_parser: 2.7.0
node: 8.9.1
v8: 6.1.534.47
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2m
icu: 59.1
unicode: 9.0
cldr: 31.0.1
tz: 2017b
os: darwin x64
5.6.0
1.3.2
```

--- a/blueprints/app/files/.eslintrc.js
+++ b/blueprints/app/files/.eslintrc.js
@@ -30,7 +30,8 @@ module.exports = {
       excludedFiles: [
         'app/**',
         'addon/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
+        'addon-test-support/**'
       ],<% } %>
       parserOptions: {
         sourceType: 'script',

--- a/tests/fixtures/addon/npm/.eslintrc.js
+++ b/tests/fixtures/addon/npm/.eslintrc.js
@@ -29,7 +29,8 @@ module.exports = {
       excludedFiles: [
         'app/**',
         'addon/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
+        'addon-test-support/**'
       ],
       parserOptions: {
         sourceType: 'script',

--- a/tests/fixtures/addon/yarn/.eslintrc.js
+++ b/tests/fixtures/addon/yarn/.eslintrc.js
@@ -29,7 +29,8 @@ module.exports = {
       excludedFiles: [
         'app/**',
         'addon/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
+        'addon-test-support/**'
       ],
       parserOptions: {
         sourceType: 'script',

